### PR TITLE
Added keras and updated README.

### PR DESCRIPTION
**Pull request by rofinn**

### CODE REVIEW

File: keras/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - h5py
+    - numpy
+    - setuptools
+    - scipy
+    - tensorflow <=1.0  # [not win]
```
Comment (iamed2) [inline keras/meta.yaml:27]: wait what

Comment (rofinn) [reply keras/meta.yaml:27]: Only tensorflow versions up to 1.0 are support for this version of keras and there were only builds for linux and osx back then. Windows builds were supported later (but possibly only on python 3.6).

Comment (iamed2) [reply keras/meta.yaml:27]: > Only tensorflow versions up to 1.0 are support for this version of keras

Do we have code somewhere which uses modern keras + tensorflow? I thought we did but maybe my memory is bad.

Comment (rofinn) [reply keras/meta.yaml:27]: Keras.jl works on the latest version, but we aren't using that package internally.

**Pull request merged**